src/main/proto/spine/options.proto

```diff
@@ -267,7 +267,10 @@ extend google.protobuf.FieldOptions {
     //
     bool column = 73854;
 
-    // Reserved 73855 to 73890 for future options.
+    // See `IfSetAgainOption`.
```

Comment:
The documentation should explain what this type is about. And then give additional information.